src/main/java/org/jabref/gui/maintable/MainTableColumnFactory.java

```diff
@@ -76,6 +78,8 @@ public MainTableColumnFactory(BibDatabaseContext database,
         this.cellFactory = new CellFactory(preferencesService, undoManager);
         this.undoManager = undoManager;
         this.stateManager = stateManager;
+        this.tooltip = new MainTableTooltip(database, dialogService, preferencesService, stateManager,
+                JabRefGUI.getThemeManager(), taskExecutor);
```

Comment:
Pleas pass the ThemeManager via constructor (Dependency Inversion Principle)

Reply:
This entails adding ThemeManager as a constructor parameter for dozen other classes. Should I still do it?

Reply:
@Siedlerchr This entails adding ThemeManager as a constructor parameter for dozen other classes. Should I still do it?

Reply:
Yes, I know this sounds crazy but we don't have really any good dependency injection. 
But most of the calling classes should already have the theme manager, so I hope it's only a handful of classes that need ajustment

Reply:
@Siedlerchr I think it's 17 files
[commit](https://github.com/JabRef/jabref/pull/11176/commits/04774ccaf8fbd7e1d7630e088aebc8069d0d51ac)

Reply:
```
ThemeManager themeManager = Injector.instantiateModelOrService(ThemeManager.class);
```
is a temporary solution until we have a proper di solution.